content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,185 @@
+  essential. This would allow easy interaction, e.g., between IRC and our chat
+  service.
+- Desktop clients and terminal-based clients were considered a great plus.
+- Ideally, this is a service that is used by other organizations as well.
+- GitHub integration and bells and whistles similar to that were considered a
+  plus as well.
+
+Using these criteria, we surveyed a bunch of options, including Mattermost,
+sticking with Slack, Discord, gitter, and of course Matrix.
+
+To test not only chat solutions, but also hosters, we rented a Mattermost and
+Matrix instance from a [Swiss hoster](https://unglei.ch) and tested them within
+a closed circle. Other services we could test with a free plan from their
+respective providers.
+
+After several weeks of trial, we conceded that Matrix ticked most of the boxes:
+
+- It is fully open-source, and committed to open standards
+- Matrix is used by many organizations such as KDE, Gnome, the Wikimedia Foundation,
+  and the French government, to name a few.
+- Matrix instances can connect to each other ("federation"). This means from our
+  GNU Radio chat, we can access other organizations on Matrix, even if they're
+  not part of the GNU Radio chat instance. This includes the main Matrix instance
+  on [matrix.org](https://matrix.org), where projects such as our friends from
+  SatNOGS are located.
+- That means that people already on Matrix don't have to do a whole lot to join
+  our Matrix server.
+- Bridging to IRC is also easy, and will have a high likelihood of working
+  indefinitely.
+- The open standards nature of Matrix, and the availability of open source
+  reference clients, means that there are lots of clients out there, including
+  things like a [Weechat plugin](https://github.com/poljar/weechat-matrix).
+
+
+## The matter of Transparency, or why am I only hearing about this now?
+
+While the GNU Radio leadership team definitely did their due diligence in
+picking a new service, one thing that definitely shouldn't become the norm for
+decisions around the GNU Radio project is the fact that we decided this behind
+closed doors within the leadership team. For example, we did not publish a GREP
+ahead of time for a public discussion.
+
+After extensive discussions, we agreed that without actually having tried it,
+the decision for a chat service would almost exclusively be a battle of
+opinions. Having all reasonable opinions among us, we decided to forgo
+fragmenting the community and instead conducted a closed testing of the most
+promising candidates.
+
+## What happens to the GNU Radio Slack?
+
+We will turn off Slack end of August 2020.
+The main development, documentation, and variety of other discussions regarding
+the project will be held on our
+Matrix instance going forward, in order to preserve them in history.
+We are paying for a one-time unlocking of the Slack history, and will make
+them available in some form to all participants of the GNU Radio chat.
```

Comment:
As mentioned on the chat because of data protection etc reasons it might be unwise to do this?

Reply:
yeah, I think publicly re-posting a chat history that people reasonably had assumed would disappear after two weeks is probably not a great plan; the fact that we might want to have these logs of internal channels (where we can reasonably assume everyone agrees to their chat messages being used for productive reasons) is another business, and we don't have to say it in a blog post.

Reply:
Agreed -- I'll remove it. It only puts us on the hook to deliver, anyway.